api/read-text.md

```diff
@@ -42,5 +48,5 @@ In this example:
 ### Example 1 Output
 
 ```json
-[[1380000000555,"test_value"],[1380000300123,"test_value_2"]]
+[[1380000000000,"test_value_at_the_beginning_of_the_requested_range"],[1380000000123,"test_value_2"]]
```

Comment:
Should these be "test" values, or "text" values?  I don't understand why we're using the word "test" here.